packages/features/form/components/LocationSelect.tsx

```diff
@@ -39,15 +43,15 @@ export default function LocationSelect(props: Props<LocationOption, false, Group
         Option: (props) => {
           return (
             <components.Option {...props}>
-              <div data-testid={`location-select-item-${props.data.value}`}>
+              <div data-testid={`location-select${idx ? `-${idx}` : ""}-item-${props.data.value}`}>
```

Comment:
Wasn't sure if we want index for all the places where LocaionSelect is used, so currently only passed from Event Type setup page so as not break other tests. Lmk if this needs to be changed